pkg/ddc/jindo/transform.go

```diff
@@ -62,8 +95,52 @@ func (e *JindoEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, value
 	return properties
 }
 
-func (e *JindoEngine) transformWorker(runtime *datav1alpha1.JindoRuntime, value *Jindo) map[string]string {
-	properties := map[string]string{}
+func (e *JindoEngine) transformWorker(runtime *datav1alpha1.JindoRuntime, dataPath string) map[string]string {
+
+	properties := map[string]string{
+		"storage.rpc.port"            : "6101",
+		//"storage.meta-dir"            : "/mnt/disk1/bigboot/bignode",
+	}
+
+	properties["namespace.meta-dir"] = dataPath + "/bigboot/bignode"
+
+	if (e.getTieredStoreType(runtime) == 0) {
+		// MEM
+		properties["storage.ram.cache.size"] = utils.TranformQuantityToJindoUnit(runtime.Spec.Tieredstore.Levels[0].Quota)
```

Comment:
need make sure  the length of `runtime.Spec.Tieredstore.Levels` is greater than 0, otherwise cause panic

Reply:
fix